Core v2 docs update (#819)
spatialos/gdk-for-unity: docs/content/workers/workers-in-the-gdk.md

This means that, along with a SpatialOS game world and SpatialOS entities, there is an ECS world with ECS entities. However, while the SpatialOS world is game-wide and represents all SpatialOS entities, the ECS world is much narrower; it’s worker-specific and represents only the ECS entities which are currently in a [worker’s view]({{urlRoot}}/content/glossary#worker-s-view).   Of course, this means that you have as many ECS worlds in your game as you have workers.

> You add definitions of the systems inside your worker using `worker.World.GetOrCreateManager<YourSystem>()`.

Reviewer: ```suggestion
You add the systems to the ECS world using `worker.World.GetOrCreateManager<YourSystem>()`.
```
